SECURITY.md

```diff
@@ -0,0 +1,55 @@
+# Security Policy
+
+## Supported Versions
+
+We commit to publishing security updates for the version of Flutter currently
+on the `stable` branch.
```

Comment:
There's no mention here of timing... is that deliberate? For example, do we aim <90 days turnaround per Project Zero?

Reply:
We presumably aim for <1 day turnaround, that is to say, as fast as possible. Why would we want to sit on a security issue? I'm not sure what it would mean to mention timing here.

Reply:
added a comment near where we mention they are P0-level.